state/state.go

```diff
@@ -44,14 +44,12 @@ type DownloadedObject struct {
 
 type DynamoDBStater struct {
 	Session          *session.Session
-	Service          string
 	BackfillInterval time.Duration
 }
 
-func NewDynamoDBStater(session *session.Session, service string, backfillHrs int) (*DynamoDBStater, error) {
+func NewDynamoDBStater(session *session.Session, backfillHrs int) (*DynamoDBStater, error) {
 	stater := &DynamoDBStater{
 		Session:          session,
-		Service:          service,
```

Comment:
I don't understand the connection between this change and the ALB stuff.  Is it going along for the ride? Is Service not useful?

Reply:
Going along for the ride, though I can split it out if you want. I noticed `Service` wasn't actually used anywhere in the `DynamoDBStater`, so passing it is not needed.